chart/templates/_helpers.yaml

```diff
@@ -248,8 +248,15 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 
 # This helper will change when customers deploy a new image.
-{{ define "airflow_image" -}}
-{{ printf "%s:%s" (.Values.images.airflow.repository | default .Values.defaultAirflowRepository) (.Values.images.airflow.tag | default .Values.defaultAirflowTag) }}
+{{- define "airflow_image" -}}
+{{- $repository := .Values.images.airflow.repository | default .Values.defaultAirflowRepository -}}
+{{- $tag := .Values.images.airflow.tag | default .Values.defaultAirflowTag -}}
+{{- $digest := .Values.images.airflow.digest -}}
```

Comment:
We should also have a `defaultAirflowDigest`, and update `default_airflow_image` too.

Reply:
That means that the chart will always use the digest rather than the tag.
I think it's preferable that when doing `kubectl describe pod` it'll show `airflow:2.5.1` rather than `airflow@sha256:...`. What do you think?

Reply:
Sorry, I meant with a default of `~`, not the digest of 2.5.1.

Reply:
All of this image config is a little confusing, but it'd retain the same behavior at least.

Reply:
Is `~` a falsely value?
I'd expect `(.Values.images.airflow.repository | default .Values.defaultAirflowRepository)` to evaluate to `("~" | default "apache/airflow")` and therefore pick `~`.

Reply:
Yes, `~` is null.